arg_postprocessing/Snakefile

```diff
@@ -329,9 +329,9 @@ rule date_internal_nodes:
     conda:
         "sc2ts.yml"
     input:
-        "{DI_PREFIX}.trees"
+        "usher_v1_2024-06-06.trees"
     output:
-        "{DI_PREFIX}_di.trees"
```

Comment:
Can you change this back please, it breaks the general usage of the command in different places. To run it, you just call 
```
snakemake usher_v1_2024-06-06_di.trees
```
Snakemake seeks the suffix "_di" and knows that it needs to find a file that matches the input for that rule. So, you can call it on any input then.

Reply:
Added a note.